api/v1alpha/api.proto

```diff
@@ -73,6 +73,9 @@ message Image {
 
         // JSON-encoded byte array that represents the image manifest, optional.
         bytes manifest = 6;
+
+	// Size of the image, (aci size) + (tree size).
+	int64 size = 7;
```

Comment:
This seems like a tab indentation while the other lines are space indentation. I don't have strong preference on either, but we should keep consistent.